packages/test-helpers/src/asserts/assertThrow.js

```diff
@@ -84,8 +86,13 @@ async function assertRevert(blockOrPromise, expectedReason, ctx) {
   }
 
   // Truffle v5 provides `error.reason`, but v4 does not.
-  if (!error.reason && error.message.includes(THROW_ERROR_PREFIX)) {
-    error.reason = error.message.replace(THROW_ERROR_PREFIX, '').trim()
+  if (!error.reason) {
```

Comment:
Doing the migration to use buidler I notice that when running tests with the buidler-evm the error is not an object and just a string like the following:
```
Error: VM Exception while processing transaction: revert INIT_ALREADY_INITIALIZED
```

As a consequence every test that is currently using our `assertRevert` fails. Wonder if we may want to consider this case?

Reply:
Yes, let's add a case for it.